### ISSUE TITLE
Add option and select option

### ISSUE THREAD

**Issue by yunusemrek32** [htmlstreamofficial/preline#320]
I want one of them to be selected after this command. Is there any way to do this?

select.addOption([
    {
      title: "James Collins",
      val: "1",
      options: {
        icon: `img`,
        selected: `true` <------------- example 
      }
    }

**Comment by dotmra** [comment]
Hey @yunusemrek32,

Are you using a single or multiple select?

If your not using multiple select, you could do something like this:
```
const select = window.HSSelect.getInstance('prelineSelect');

// Destroy and reinitialize the select.
select.destroy();

// Add options (if you have an array of values/options) otherwise just create a single option and append it.
values.forEach(value => {
    const option = new Option(
        'James Collins', // value.name,
        '4', // value.id,
        false, // This is if the options is default selected or not. 
        false // value.selected
    );
    
    document.getElementById('prelineSelect').appendChild(option);
});

new HSSelect('prelineSelect');
```

If it is a multiple select, I can't help currently, see #335

**Comment by jahaganiev** [comment]
Hey @yunusemrek32, @dotmra - our team just released version 2.4.0, which now includes this option.
